lib/java/deploy/MavenPerBranchSpringBootDeploymentGoal.ts

```diff
@@ -34,6 +35,31 @@ import * as os from "os";
 import * as portfinder from "portfinder";
 import { MavenLogInterpreter } from "../../maven/build/mavenLogInterpreter";
 
+export const ListBranchDeploys: CommandHandlerRegistration = {
+    name: "listLocalDeploys",
+    intent: "list branch deploys",
+    description: "list branch deploys",
+    listener: async ci => handleListDeploys(ci.context),
+};
+
+async function handleListDeploysWith(description: string, ctx: HandlerContext) {
```

Comment:
It does not seem like the `description` is used so it is not clear why you have separated this into two functions.

Reply:
Left over by the `list local deploys` on which this was based. Removed.